extensions/clippyx/package.json

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "clippyx",
+  "title": "CLIPPyX",
+  "description": "CLIPPyX provides an OS-wide image search that supports semantic search in both image content and text on images",
+  "icon": "CLIPPyX.png",
+  "author": "0ssamaak0",
+  "categories": [
+    "Applications"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "caption",
+      "title": "Caption Search",
+      "description": "Search images by caption",
+      "mode": "view",
+      "preferences": [
+        {
+          "name": "threshold",
+          "type": "textfield",
+          "required": true,
+          "title": "Threshold",
+          "description": "Float value between 0 and 1",
+          "placeholder": "0.2",
+          "default": "0.2"
+        },
+        {
+          "name": "topK",
+          "type": "textfield",
+          "required": true,
```

Comment:
Would it make sense to set all `required` to `false`  instead so the user don't need to set them before using each command?

Reply:
I think it's better to leave them `true`? they will be set in the first time only right?